### ISSUE TITLE
[Sourcing][P3.1][Accessibility] `fdp-menu-button` lost focus after close the menu

### ISSUE THREAD

**Issue by oscargong** [SAP/fundamental-ngx#11398]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Use keyboard, after close `fdp-menu-button`, the focus back to browser URL bar

What is expected:
the visible focus should always follow exact keyboard commands and the keyboard focus should be always present on the same UI from where it was activated or closed and never be lost.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
```
"@fundamental-ngx/cdk": "0.48.4",
  "@fundamental-ngx/platform": "0.48.4",
  "@fundamental-ngx/core": "0.48.4",
  "@fundamental-ngx/i18n": "0.48.4",
```

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

https://github.com/SAP/fundamental-ngx/assets/7035787/0d6d008b-e825-400d-a9ac-206e2fa42534

#### Please provide relevant source code (if applicable).
https://sap.github.io/fundamental-ngx/#/platform/menu-button

#### Please provide stackblitz example(s).
https://sap.github.io/fundamental-ngx/#/platform/menu-button

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes I did

**Comment by oscargong** [comment]
this is happening on V0.48.4, but we are using 0.43 on our application, need a fix on ng-15-downport
